bundles/org.openhab.binding.sony/src/main/resources/ESH-INF/config/dial.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<config-description:config-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0" xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:sony:dialconfig">
+		<parameter name="deviceAddress" type="text" required="true">
+			<context>network-address</context>
+			<label>Dial URL</label>
+			<description>The URL access point for the DIAL service</description>
+		</parameter>
+		<parameter name="deviceMacAddress" type="text">
+			<label>MAC Address</label>
+			<description>The MAC address of the device</description>
+		</parameter>
+		<parameter name="accessCode" type="text">
+			<label>Access Code</label>
+			<description>The access code (or "RQST" to request one)</description>
+			<default>RQST</default>
+		</parameter>
+		<parameter name="refresh" type="integer">
+			<label>Refresh Interval</label>
+			<description>The interval, in seconds, to refresh the device state (-1 to disable)</description>
+			<default>30</default>
+			<advanced>true</advanced>
+		</parameter>
+		<parameter name="retryPolling" type="integer">
+			<label>Retry Polling</label>
+			<description>The time, in seconds, to retry a connection attempt (-1 to disable)</description>
+			<default>10</default>
+			<advanced>true</advanced>
+		</parameter>
+		<parameter name="checkStatusPolling" type="integer">
+			<label>Status Check Interval</label>
+			<description>The time, in seconds, to check the device status (-1 to disable)</description>
+			<default>30</default>
+			<advanced>true</advanced>
```

Comment:
What is the difference between the device state and the device status?

Reply:
There are three different pollings:
1. Device status - the time between checking to see whether we have a valid connection to the device.  If we can't connect, brings the thing offline and starts a reconnection attempt
2. Device state - the time between checking the state of the device.  This will take the current state and update all associated channels.  Necessary if the user is making changes to the device outside of openHAB (like from a remote)
3. Connection poll - the time between connection attempts.  Basically attempts to bring the thing back ONLINE.

I'll take this review comment as a need to update the README.md on these options to make them more clear